src/components/CardEdition/CardEdition.tsx

```diff
@@ -0,0 +1,60 @@
+import * as React from "react";
+import bem from "../../utils/bem";
+
+interface CardEditionProps {
+  /** ClassName that appears in addition to "card" */
+  className?: string;
+  /** BlockName for use with BEM. See how to work with blockNames and BEM here: http://getbem.com/introduction/ */
+  blockName?: string;
+  /** Calls to Action for the card, such as "download" on a card for a book */
+  ctas?: React.ReactNode;
+  /** Footer that appears within the card but below the three columns within it */
+  footer?: React.ReactNode;
+  /** Heading that appears within the card but above the three columns within it */
+  heading?: React.ReactNode;
+  /** ID that other components can cross reference for accessibility purposes */
+  id?: string;
+  /** First column within the card */
+  image?: React.ReactNode;
+  /** Modifiers array for use with BEM. See how to work with modifiers and BEM here: http://getbem.com/introduction/ */
+  modifiers?: string[];
+}
+
+export default function CardEdition(
```

Comment:
Is this suppose to be the "legacy" or "deprecated" component for `Card`? Or is it suppose to be a different component?

Reply:
`CardEdition` will be deprecated.